doc/rtd/reference/custom_modules/custom_mergers.rst

```diff
@@ -0,0 +1,66 @@
+.. _custom_mergers:
+
+Custom Mergers
+**************
+
+TODO adapt and backreference
+
+Because the default
+:ref:`merging<merging_user_data>`
+algorithms and stragies may not always be desired,
+the concept of customised merging was introduced through `merge classes`.
```

Comment:
Nit: now that we've switched back to American English
```suggestion
the concept of customized merging was introduced through `merge classes`.
```

Reply:
I've removed this phrase in favor of the existent overview that was down.